news.html

```diff
@@ -3,44 +3,44 @@
     <div>
       <h5 class="mb-0">
           <a target="_blank" rel="noopener noreferrer"
-             href="https://github.com/dockstore/dockstore/releases/tag/1.14.0">
-              1.14 release</a>
+             href="https://github.com/dockstore/dockstore/releases/tag/1.15.0">
```

Comment:
Tag doesn't exist, but will

Reply:
Looks like in the past, we've updated by replacing the previous release announcement with the new release announcement, so this PR conforms.  Which is fine, except it has the effect of eliminating the one existing news item from 2023, making us appear less active than we actually are...

Reply:
Yeah, maybe dropping the oldest makes more sense

Reply:
Restored 1.14 news, dropping oldest.

I'll wait to merge this until we have a release date.